server/controllers/accountController.test.js

```diff
@@ -0,0 +1,172 @@
+import mongoose from 'mongoose';
+import request from 'supertest';
+import express from 'express';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+import accountController from './accountController.js';
+import Account from '../../database/models/account.js';
+import Customer from '../../database/models/customer.js';
+
+dotenv.config();
+
+const app = express();
+app.use(bodyParser.json());
+
+app.get('/api/accounts', accountController.getAllAccounts);
+app.post('/api/accounts', accountController.createAccount);
+app.get('/api/accounts/customer/:id', accountController.getAllCustomerAccounts);
+app.put('/api/accounts/:id', accountController.updateAccount);
+app.delete('/api/accounts/:id', accountController.deleteAccount);
+
+jest.setTimeout(30000); // Increase Jest timeout to 30 seconds
+
+describe('Account Controller', () => {
+  beforeAll(async () => {
+    await mongoose.connect(process.env.DB_URI);
```

Comment:
The controller tests are connecting to the live DB. I think you need to youse MongoMemoryServer in these files similar to what you have done in the models tests.

Reply:
Can you also update the database/config/helpers.test.js to include the below as it's causing an error with not being able to find the DB_URI string.

`import dotenv from 'dotenv';

dotenv.config();`